src/operator/controllers/aws_iam/spiffe_rolesanywhere_pod_webhook/spiffe_pod_webhook.go

```diff
@@ -0,0 +1,253 @@
+package spiffe_rolesanywhere_pod_webhook
```

Comment:
The new webhook that creates SPIFFE identities with the csi-driver and works with RolesAnywhere.